modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/reconnection/ReconnectionTestCase.java

```diff
@@ -88,6 +89,23 @@ public void reconnectSource() throws Exception {
     });
   }
 
+  @Test
+  public void doNotStartSourceTwiceAfterExceptionOnReconnection() throws Exception {
+    ((Startable) getFlowConstruct("otherReconnectForever")).start();
```

Comment:
this test depends greatly on `ReconnectableConnectionProvider.otherFail`. The initial value of that flag should be set of `doSetUp()`

Reply:
Done